docs/developer/README.md

```
@@ -168,12 +168,14 @@ make agent-container
 make clean-agent-container
 ```
 
-Use the **[devicesimulator](devicesimulator.md)** to simulate load from devices
+Use the **[devicesimulator](devicesimulator.md)** to simulate load from devices. Preferred entrypoint:
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)